config/hero.yaml

```
@@ -7,7 +7,7 @@ buttons:
 shortcuts:
   - title: Curious about the basics?
     description: |
-      AEPs are a combination of design guidance and a system we use to manage
+      AEPs are a combination of design guidance and a system to manage
       and track that guidance. Learn more about how the AEP program works in
       the first AEP!
```

```suggestion
      AEPs are a collection of API design specifications, design patterns, guidance. In addition, the AEP project includes tools to help produce AEP-compliant APIs, as well a clients (command-lines, UIs, Declarative plugins) that consume them. Learn more about how the AEP program works in the first AEP!
```

It's a mouthful I know, but I think it's really important to emphasize that:

1. this is a specification and not just design guidance.
2. we're building a whole ecosystem of tooling and not just the design guidance itself.

Thoughts? it would also be great to get the grpc / oas (http+json) part too, but I know that's getting really long...